src/react-image-lightbox.js

```diff
@@ -1759,6 +1803,8 @@ ReactImageLightbox.propTypes = {
   zoomInLabel: PropTypes.string,
   zoomOutLabel: PropTypes.string,
   closeLabel: PropTypes.string,
+
+  imageLoadErrorMessage: PropTypes.string
```

Comment:
I think `PropTypes.node` would be more versatile. That way they could plug in any element they wanted to in that area.